src/algebra/algebra/subalgebra.lean

```diff
@@ -416,6 +416,15 @@ def equalizer (ϕ ψ : A →ₐ[R] B) : subalgebra R A :=
 @[simp] lemma mem_equalizer (ϕ ψ : A →ₐ[R] B) (x : A) :
   x ∈ ϕ.equalizer ψ ↔ ϕ x = ψ x := iff.rfl
 
+--/-- Range of an `alg_hom` as a subalgebra. -/
+--protected def range (φ : A →ₐ[R] B) : subalgebra R B :=
+--{ algebra_map_mem' := λ r, ⟨algebra_map R A r, φ.commutes r⟩,
+--  .. φ.to_ring_hom.srange }
```

Comment:
This should be deleted, I assume it's already defined elsewhere

Reply:
yes, it's about 50 lines up.